tests/dependencies.py

```diff
@@ -0,0 +1,12 @@
+import sys
+import pkg_resources
+
+
+def test_tensorflow_text_install():
```

Comment:
There is probably a better place for this to go, this isn't really a unit test so it shouldn't be in `tests`. I just added it here for now so I can see in the CI if it works on both Linux and on Windows.